src/lib/djerba/plugins/supplement/body/plugin.py

```diff
@@ -63,6 +63,12 @@ def extract(self, config):
         else:
             report_signoff_date = wrapper.get_my_string(self.REPORT_SIGNOFF_DATE)
         self.check_assay_name(wrapper)
+       
+        if wrapper.get_my_attributes()[0] == "clinical":
```

Comment:
- Better to have `if "clinical" in wrapper.get_my_attributes()` -- at some point we might define other attributes, and the order is not guaranteed
- Similarly for "research"